.github/workflows/ci.yml

```diff
@@ -445,7 +445,7 @@ jobs:
           sudo apt-get update
           sudo apt-get -y install libsqlite3-dev libpq-dev libmysqlclient-dev
       - name: Check diesel_derives
-        run: cargo +1.65.0 minimal-versions check -p diesel_derives --all-features
+        run: cargo +1.65.0 minimal-versions check -p diesel_derives --features "postgres sqlite mysql 32-column-tables 64-column-tables 128-column-tables without-deprecated with-deprecated r2d2"
```

Comment:
Because `--all-features` would enable the `nightly` feature for `diesel-derives`, which in turn would enable an unstable rustc feature, which in turn will not compile on a stable release.
The feature list is equivalent to `--all-features` with the `nightly` feature removed.